### ISSUE TITLE
[a11y] Relay homepage missing text labels for graphics

### ISSUE THREAD

**Issue by rbillings** [mozilla/fx-private-relay-add-on#247]
The Relay homepage is missing text labels for graphics. This makes it extremely difficult to navigate the page with a screenreader, and does not meet WCAG accessibility standards.

URL = https://relay.firefox.com/premium

Graphics list:
3 perk-illustration elements, name = null
2 c-brand-title elements, name = null

**Comment by Vinnl** [comment]
Will be fixed by #1452 when that's deployed.
